config/routes.rb

```diff
@@ -1,2 +1,4 @@
 Rails.application.routes.draw do
-end
+  resources :products
```

Comment:
you need to limit what routes you have for products to what you actually use atm in the app

e.g.
`resources :products, only: [:index]`